frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-subscribe.tsx

```diff
@@ -334,6 +373,16 @@ export const OperatorHubSubscribeForm: React.FC<OperatorHubSubscribeFormProps> =
         await k8sCreate(OperatorGroupModel, operatorGroup);
       }
       await k8sCreate(SubscriptionModel, subscription);
+      if (!_.isEmpty(enabledPlugins)) {
+        const previousPlugins: string[] = consoleOperator?.spec?.plugins || [];
+        await k8sPatch(ConsoleModel, consoleOperator, [
+          {
+            path: '/spec/plugins',
+            value: _.union(previousPlugins, enabledPlugins),
+            op: 'add',
```

Comment:
I'm debating if we need to remove existing enabled plugins if we are reinstalling an operator and the user picks "Disabled."

Reply:
> I'm debating if we need to remove existing enabled plugins if we are reinstalling an operator and the user picks "Disabled."

I think the logic for this is pretty simple and we should go ahead and do it. Something like

```ts
value: [
  ...previousPlugins.filter((plugin: string) => !csvPlugins.includes(plugin)),
  ...enabledPlugins,
],
```

Reply:
This does not work as anticipated since `op: 'add'`.  I think we need to `op: 'remove'` as a separate condition?

Reply:
Hm, what happens? I'd think we do want add, which works as either add or replace depending on whether the array exists. Here we're replacing the entire array. But I haven't tested.

Reply:
I think https://github.com/openshift/console/pull/8175/commits/f4db0cb36e51b77e875f3a77d4b3c262eee52aeb is what we want, @spadgett?

Reply:
Updated in be8d619ae2a06f013d786446acfe0fd491433c35